### PR TITLE
[LIWO-548] Merge imported combine scenarios

### PR DESCRIPTION
**Pull request by petergoes**
This adds the option to merge the existing selection with the imported selection.

Rijkswaterstaat uses native radiobuttons in their stylesheet. So I used the native checkbox as well since checkboxes are not documented on the styleguide of Rijkswaterstaat

### CODE REVIEW

File: public/config/webconfig.json [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
   "_comment0": "This file will be overwritten automatically by a puppet script in the deployment process, DO NOT EDIT BY HAND, DO NOT MINIFY",
   "_comment1": "url's for webservice calls",
-  "WEBSERVICE_URL": "http://tw-160.xtr.deltares.nl/liwo.ws",
-  "STATIC_GEOSERVER_URL": "http://tl-396.xtr.deltares.nl:8080/geoserver/ows",
-  "LEGEND_URL": "http://tl-396.xtr.deltares.nl:8080/geoserver/ows",
-  "DYNAMIC_GEOSERVER_URL": "http://tl-397.xtr.deltares.nl:8080/geoserver/ows",
+  "WEBSERVICE_URL": "https://basisinformatie-overstromingen.nl/liwo.ws",
+  "STATIC_GEOSERVER_URL": "https://geodata.basisinformatie-overstromingen.nl/geoserver/ows",
+  "LEGEND_URL": "https://geodata.basisinformatie-overstromingen.nl/geoserver/ows",
+  "DYNAMIC_GEOSERVER_URL": "https://profgeodata.dynamic.basisinformatie-overstromingen.nl/geoserver/ows",
```
Comment (SiggyF) [inline public/config/webconfig.json:7]: please don't check in this file.

File: src/components/ImportCombinePopUp.vue [hunk 2]
```diff
@@ -66,9 +80,15 @@ export default {
       if (!match) {
         this.showError = true
       } else {
+        let newIds = match[0].split(',')
+        let currentIds = this.currentSelectedIds.split(',')
+        if (this.combineWithCurrentSelectedIds) {
+          newIds = _.uniq([...currentIds, ...newIds].filter(x => x))
```
Comment (SiggyF) [inline src/components/ImportCombinePopUp.vue:86]: What does filter x=> x do?

**Review by SiggyF — APPROVED** [review]
Looks good, can you restore the old webconfig.json file?

File: src/components/ImportCombinePopUp.vue (hunk 2 above)
Comment (petergoes) [reply src/components/ImportCombinePopUp.vue:86]: `x => x` is a function which returns the given parameter. Also known as an [identity](https://en.wikipedia.org/wiki/Identity_function) function.

It is used here to filter out `falsy` values.
`filter` loops over all items in an array. It accepts a function, which should return `true` or `false`.  When `true` the array item is kept, when `false`, the array item is discarded.

Imagine the array `["", "one", "two"]`. 
When looped over with `filter` (`["", "one", "two"].filter(x => x)`), the first time, the function given to filter (`x => x`) gets `""` as its argument. The function returns it. `filter()` will [coerce](https://2ality.com/2019/10/type-coercion.html) it into a Boolean. `Boolean("")` will become `false`, thus, the fist item of the array is discarded.
The second and third time, the result of `Boolean("one")` and `Boolean("two")` will be `true`, thus, these values will remain.

So: `console.log(["", "one", "two"].filter(x => x)) // ["one", "two"]`

The remaining array is passed to Lodash's `uniq` function which will remove all duplicates. This results in an array without falsy, unique values

File: public/config/webconfig.json (hunk 1 above)
Comment (petergoes) [reply public/config/webconfig.json]: I let it slip, sorry

File: src/components/ImportCombinePopUp.vue (hunk 2 above)
Comment (SiggyF) [reply src/components/ImportCombinePopUp.vue:86]: We might have an id: 0. Will that scenario be removed?

**Comment by SiggyF** [comment]
CI / build (pull_request) Failing will be fixed in another branch. SIG deploy is moved to teamcity.

File: src/components/ImportCombinePopUp.vue (hunk 2 above)
Comment (petergoes) [reply src/components/ImportCombinePopUp.vue:88]: it will

Comment (petergoes) [reply src/components/ImportCombinePopUp.vue:88]: No, it will not. the id's will always be strings, so the value will be: `"0"`.  That is not falsy

**Pull request merged**